### PR TITLE
tag: Add an input for redirect page name for R avoided double redirect

### PR DESCRIPTION
**Pull request by TheTVExpert**
Added an input to allow the user to enter the redirect page name for the R avoided double redirect template.

### CODE REVIEW

File: modules/friendlytag.js [hunk 1]
```diff
@@ -1671,6 +1680,9 @@ Twinkle.tag.callbacks = {
 				if (params.altLangTo) {
 					tagText += '|to=' + params.altLangTo;
 				}
+				if (params.doubleRedirectTarget) {
+					tagText += '|1=' + params.doubleRedirectTarget;
+				}
 			}
```
Comment (Amorymeltzer) [inline modules/friendlytag.js:1686]: This is an indent too deep; right now it's under `(tagName === 'R from alternative language')`, which obviously won't work!

What should happen is something like:

```suggestion
				if (params.altLangTo) {
					tagText += '|to=' + params.altLangTo;
				}
			} else if (tagName === 'R avoided double redirect' && params.doubleRedirectTarget) {
				tagText += '|1=' + params.doubleRedirectTarget;
			}
```

**Review by Amorymeltzer — CHANGES_REQUESTED** [review]
Needs a quick bug fix, but otherwise nice PR @TheTVExpert!

**Comment by TheTVExpert** [comment]
Fixed.

**Review by Amorymeltzer — APPROVED** [review]
Looks great, nice addition!  I wonder how many others there are that might benefit from this kind of thing?

**Comment by Amorymeltzer** [comment]
Merged as 2c31888

**Pull request closed without merging**